plugins/MantisGraph/MantisGraph.php

```diff
@@ -113,6 +113,7 @@ function resources() {
 			html_javascript_link( 'chart-' . CHARTJS_VERSION . '.min.js' );
 			html_javascript_link( 'chart.bundle-' . CHARTJS_VERSION . '.min.js' );
 		}
+		html_javascript_link( '../plugins/MantisGraph/MantisGraph.js' );
```

Comment:
Move the MantisGraph.js file into plugins/MantisGraph/files and include it by getting its location using plugin_file( 'MantisGraph.js' ).  See Snippets plugin for an example.